app/views/admin/roles/edit.html.erb

```diff
@@ -1,21 +1,34 @@
-<% page_title "Edit #{@role.name}"  %>
-<div class="row">
-  <section class="col-md-8">
-    <h1>Edit role</h1>
-    <p class="warning">Warning: changes to roles appear instantly on the live site.</p>
+<% content_for :page_title, "Edit #{@role.name}" %>
+<% content_for :title, "Edit #{@role.name}" %>
+<% content_for :title_margin_bottom, 6 %>
+
```

Comment:
this needs the error summary banner added here please 

```
<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @role)) %>

```

Reply:
done. cool, thanks. Don't think I'd quite twigged how the error handling works. Got a better sense of it now 👍